app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -105,3 +106,10 @@ export default class SingleAddon extends React.Component{
     );
   }
 }
+
```

Comment:
Same question as above.

Reply:
@dkayiwa The person who wrote the component I was testing didn't add required props validation, so I added it.